common/types.ts

```diff
@@ -155,3 +153,14 @@ export type StatusApiResponse = {
 export type TeamApiResponse = Team;
 
 export type DatesApiResponse = string;
+
+export type ApplicantsApiResponse = {
+  data: Array<User & { _id: string }>;
```

Comment:
just to explain this: basically we don't want `_id` to always exist on the type user, we just want it to exist for these specific two endpoints (the ApplicantsApiResponse, SingleApplicantApiResponse). so, just add it where it's needed. (the `&` just makes a sum type)